.github/workflows/build.yaml

```diff
@@ -12,12 +12,39 @@ on:
 
 env:
   REGISTRY: ghcr.io
-  NAMESPACE: galasa-dev
+  NAMESPACE: ${{ github.repository_owner }}
+  BRANCH: ${{ github.ref_name }}
 
 jobs:
+  check-secrets:
+    # Run this job for forks
+    if: ${{ github.repository_owner != 'galasa-dev' }}
+    name: Check for required secrets
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Check if WRITE_GITHUB_PACKAGES_USERNAME is set in this repository
+        run: |
+          if [ -z "${{ secrets.WRITE_GITHUB_PACKAGES_USERNAME }}" ]; then
+            echo "WRITE_GITHUB_PACKAGES_USERNAME is not set. Please configure it in the repository secrets."

```

Comment:
Would it make sense to provide more of an explanation as to what `WRITE_GITHUB_PACKAGES_USERNAME` and `WRITE_GITHUB_PACKAGES_TOKEN` should be set to in the respective error messages for clarity?

Reply:
I have now added more detail to these comments explaining what the secrets need to contain.